python/lsst/ts/m2gui/controltab/tab_detailed_force.py

```diff
@@ -33,7 +50,258 @@ class TabDetailedForce(TabDefault):
         Table's title.
     model : `Model`
         Model class.
+
+    Attributes
+    ----------
+    model : `Model`
+        Model class.
     """
 
+    # These are contants used in the self._resize_table() before figuring out
```

Comment:
contants -> constants

Reply:
Fixed in the commit: [7127817](https://github.com/lsst-ts/ts_m2gui/pull/8/commits/7127817f46a8471c10748ab039065b7f9df0c9b1). Thanks!